client/utils/account-fees.tsx

```diff
@@ -77,7 +77,7 @@ const getFeeDescriptionString = ( fee: BaseFee ): string => {
 	return '';
 };
 
-export const getCurrentFee = (
+export const getCurrentBaseFee = (
```

Comment:
Maybe a nitpick. I find the name of this function a bit misleading. Looking at the return type it can return BaseFee or DiscountFee so I'd suggest changing its name to reflect that: `getCurrentBaseFeeOrDiscountFee`.

You could also argue that we have return types so we don't have to rely on function names to tell what a function will return. But in that case, I would leave it as before `getCurrentFee`.

Reply:
This client part will need some updates as we finish migrating fees into DB. Hence we'll make another round of naming nitpicking.

Reply:
Thanks for pointing this out, Miguel. From what I spotted, as seen by the merchant in the settings page: 

1. When there is a discount rate available, The base fee is shown as the discounted price 
2. When there is no discount rate available, the base fee is shown as the default ( without discount override ). 

The merchant-facing label, in either case remains as `Base Fee`.  This is my understanding. Happy to revisit this if needed. Thanks for the review!